integration_tests/test_array_01.py

```diff
@@ -0,0 +1,21 @@
+from ltypes import i32, f64
+
+def test_array_01():
+    x: f64[3] = [1.0, 2.0, 3.0]
```

Comment:
This approach is not consistent with CPython:
In CPython, the variable `x` will be of type `list`. But here as can be seen it is of type `array`!